spec.bs

```diff
@@ -676,3 +683,190 @@ To <dfn>determine the numeric value</dfn> of a [=signal base value=]
+### Cross-network reporting origin leakage ### {#cross-network-reporting-origin-leakage}
+
+A report may be stored while the browser is connected to one network but sent
+while the browser is connected to a different network, potentially enabling
+cross-network leakage of the reporting origin.
+
+Example: A user runs the browser with a particular browsing profile on their
+home network. An aggregatable report with a particular reporting origin is
+stored with a report time in the future. After the report time is reached, the
+user runs the browser with the same browsing profile on their employer’s
+network, at which point the browser sends the report to the reporting origin.
+Although the report itself may be sent over HTTPS, the reporting origin may be
+visible to the network administrator via DNS or the TLS client hello (which can
+be mitigated with ECH). Some reporting origins may be known to operate only or
+primarily on sensitive sites, so this could leak information about the user’s
+browsing activity to the user’s employer without their knowledge or consent.
+
+Possible mitigations include:
+
+1. Only sending reports with a given reporting origin when the browser has
+    already made a request to that origin on the same network: This prevents the
+    network administrator from gaining additional information from the Private
+    Aggregation API. However, it increases report loss and report delays, which
+    reduces the utility of the API for the reporting origin. It might also
+    increase the effectiveness of timing attacks, as the origin may be able to
+    better link the report with the user’s request that allowed the report to be
+    released.
+1. Send reports immediately: This reduces the likelihood of a report being
+    stored and sent on different networks. However, it increases the likelihood
+    that the reporting origin can correlate the original API invocation to the
+    report being sent, which weakens the privacy controls of the API, see
+    [Protecting against leaks via the number of
+    reports]{#protecting-against-leaks-via-the-number-of-reports}.
+1. Use a trusted proxy server to send reports: This effectively moves the
+    reporting origin into the report body, so only the proxy server would be
+    visible to the network administrator.
+1. Require [DNS over HTTPS](https://datatracker.ietf.org/doc/html/rfc8484): This
+    effectively hides the reporting origin from the network administrator, but
+    is likely impractical to enforce and is itself perhaps circumventable by the
+    network administrator, e.g. by monitoring IP addresses instead.
+
+### User-presence tracking ### {#user-presence-tracking}
+
+The browser only tries to send reports while it is running and while it has
+internet connectivity (even without an explicit check for connectivity,
+naturally the report will fail to be sent if there is none), so receiving or not
+receiving an aggregatable report at the expected time leaks information about
```

Comment:
The aggregatable report is delayed by a random time, so the "expected time" may be a bit unclear?